tmail-backend/tmail-third-party/ai-bot/src/main/scala/com/linagora/tmail/jmap/aibot/AIChatCompletionRoutes.scala

```diff
@@ -81,12 +80,18 @@ class AIChatCompletionRoutes @Inject()(@Named(InjectionKeys.RFC_8621) val authen
         .aggregate()
         .asByteArray())
       .flatMap(input => SMono.fromPublisher(openRagHttpClient.proxyChatCompletions(input)))
-      .flatMap(result => SMono.fromPublisher(response.status(result.status())
-            .headers(result.headers())
```

Comment:
I would have thought this was enough. Seems not in the end... thanks for the fix!